implementation/src/main/java/io/smallrye/config/DefaultValuesConfigSource.java

```diff
@@ -1,14 +1,14 @@
-package io.smallrye.config.mapper;
+package io.smallrye.config;
 
 import java.util.Collections;
 import java.util.Map;
 
 import org.eclipse.microprofile.config.spi.ConfigSource;
 
-final class DefaultValuesConfigSource implements ConfigSource {
+public final class DefaultValuesConfigSource implements ConfigSource {
```

Comment:
I would rather not see this exposed in this way; rather I think it would be better to introduce an API to `SmallRyeConfigBuilder` to add default values directly, and let the mapper use that API.  I believe this is similar to a proposal I sent to the MP Config project.

Reply:
This was just am intermediary step with the integration, since mapping had its own package.

Right now, this was already removed in favour of:
https://github.com/smallrye/smallrye-config/commit/b0b61c2f44a84d84275faa740198a234207ab2f8

Let me know if this lines up better with what you were thinking for default values.